doc/source/ray-core/doc_code/fault_tolerance_tips.py

```diff
@@ -0,0 +1,77 @@
+# __return_ray_put_start__
+import ray
+
+# Non-fault tolerant version:
+@ray.remote
+def A():
+    x_ref = ray.put(1)
+    return x_ref
+
+x_ref = ray.get(A.remote())
+# Object x outlives its owner task A.
+try:
+    # If owner of x (i.e. the worker process running task A) dies,
+    # the application can no longer get value of x anymore.
+    print(ray.get(x_ref))
+except ray.exceptions.OwnerDiedError:
+    pass
+
+
+# Fault tolerant version:
```

Comment:
It would be better to break this up into a second code block and interleave it with the explaining text, so that the "caption" is closer to the example.